source/appModuleHandler.py

```diff
@@ -51,6 +53,75 @@
 post_appSwitch = extensionPoints.Action()
 
 
+_EXECUTABLE_NAMES_TO_APP_MODS_CORE: Dict[str, str] = {

```

Comment:
This dictionary might be better placed in `appModules/__init__.py` - just to be closer to the core app modules.
For a developer adding a new core appModule, checking `appModules/__init__.py` is more intuitive.
It also separates the "data" from this code file, and slows the file length growth of `source/appModuleHandler.py`.